### PR TITLE
OpenSearch  security - Adding section on updating OpenSearch

### PR DESCRIPTION
**Pull request by SeppeDejonckheere**

### CODE REVIEW

**Comment by MariekeGO** [comment]
@SeppeDejonckheere, I've reviewed the PR and made some changes. Could you check if everything is still OK for you?

File: user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Opensearch.md [hunk 1]
```diff
@@ -10,3 +11,45 @@ uid: Security_OpenSearch
...
+There are two ways to perform a minor upgrade of an OpenSearch cluster:
+
+- [A cluster restart upgrade](#cluster-restart-upgrade)
+- [A rolling upgrade](#rolling-upgrade)
+
+Regardless of which upgrade strategy you choose, you will first need to perform these steps:
+
+1. Download the latest version within your release track: [1.X](https://opensearch.org/lines/1x.html) or [2.X](https://opensearch.org/lines/2x.html).
+1. [Back up the configuration files](https://opensearch.org/docs/latest/install-and-configure/upgrade-opensearch/index/#backing-up-configuration-files).
+1. Back up the cluster by [taking a snapshot](https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore/).
```
Comment (MariekeGO) [inline user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Opensearch.md:32]: We have a procedure for taking snapshots within our documentation: https://docs.dataminer.services/user-guide/Advanced_Functionality/Databases/Indexing_database/OpenSearch_Database/Configuring_OpenSearch_Backups.html#taking-the-snapshot
Could we link to that instead?

Comment (SeppeDejonckheere) [reply user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Opensearch.md:32]: I was not aware we already had this documented, I'll make the necessary changes

**Comment by SeppeDejonckheere** [comment]
Updated the backup reference. The other changes look fine to me.

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**